scripts/mdbook-forc-documenter/README.md

```diff
@@ -0,0 +1,33 @@
+# mdbook-forc-preprocessor
+
+A preprocessor for [mdBook](https://github.com/rust-lang/mdBook) to update the Forc commands section of the Sway book based on the output we get when running `forc --help`.
+
+This preprocessor is automatically run on every build, as long as the `book.toml` file contains the preprocessor:
+
+```toml
+[preprocessor.forc-documenter]
+```
+
+## Usage
+
+### Adding a new forc command
+
+Enter a new entry under the `Commands` section within `SUMMARY.md`, in this format:
+
+```md
+- [forc gm](./forc_gm.md)\n
```

Comment:
```suggestion
- [forc gm](./forc_gm.md)
```
Does it need the `\n` on the end like this?

Reply:
Nope! Thanks for the catch